sample/src/main/java/com/github/squti/androidwaverecordersample/MainActivity.kt

```diff
@@ -173,6 +186,17 @@ class MainActivity : AppCompatActivity() {
         private const val TAG = "MainActivity"
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
```

Comment:
This is not something to change right now. But in newer versions of Android onActivityResult is deprecated, and in future versions it may be deleted at all. There is a new API for that **ActivityResultCallback**.